---
string hair_length
string hair_color
